src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -109,6 +111,8 @@ private void initialize() {
                     });
                     addToggle.getStyleClass().add("addEntryButton");
                     addToggle.selectedProperty().bindBidirectional(entriesListView.getItemBooleanProperty(entry));
+                    addToggle.selectedProperty().addListener((observable, oldValue, newValue) ->
```

Comment:
Is it not possible to bind the selectedItems and totalItems text properties? E.g.
```
selectedItems.textProperty().bind(Bindings.size(entriesListView.getCheckModel().getCheckedItems()).asString())
```

Reply:
It is possible. Thanks.